wallet/db.c

```diff
@@ -596,6 +596,9 @@ static struct migration dbmigrations[] = {
      * Turn anything in transition into a WIRE_TEMPORARY_NODE_FAILURE. */
     {SQL("ALTER TABLE channel_htlcs ADD localfailmsg BLOB;"), NULL},
     {SQL("UPDATE channel_htlcs SET localfailmsg=decode('2002', 'hex') WHERE malformed_onion != 0 AND direction = 1;"), NULL},
+    /* For incoming HTLCs, we now keep track of whether or not we provided
+     * the preimage for it, or not. */
+    {SQL("ALTER TABLE channel_htlcs ADD we_filled INT;"), NULL},
```

Comment:
`sqlite3` is very lax with regards to its type-system, they're effectively aliases. In `postgres` this would be an `s32`, so we need to make sure that doesn't break.

Reply:
this has been addressed. @rustyrussell was there another comment that was overlooked?